packages/graphics/bcm2835-driver/package.mk

```diff
@@ -45,6 +45,9 @@ make_target() {
     ln -sf $SYSROOT_PREFIX/usr/lib/libbrcmEGL.so $SYSROOT_PREFIX/usr/lib/libEGL.so
     ln -sf $SYSROOT_PREFIX/usr/lib/libbrcmGLESv2.so $SYSROOT_PREFIX/usr/lib/libGLESv2.so
     cp -PRv $FLOAT/opt/vc/lib/*.a $SYSROOT_PREFIX/usr/lib
+
+  mkdir -p $SYSROOT_PREFIX/usr/lib/pkgconfig
+    cp -PRv $FLOAT/opt/vc/lib/pkgconfig/* $SYSROOT_PREFIX/usr/lib/pkgconfig
```

Comment:
@lrusak @MilhouseVH : yeah will update it no problem, just have been away from home for work. I'll update this PR on monday :)